addons/60fps/addon.json

```diff
@@ -6,6 +6,11 @@
       "type": "notice",
       "text": "Changing the framerate of Scratch projects also affects the rate at which certain code (such as loops) runs, so most projects won't behave properly when run at a different framerate. Higher framerates are only meant to be used on projects that are designed to support them.",
       "id": "projectRunsFasterNotice"
+    },
+    {
+      "type": "notice",
+      "text": "If your browser is in Energy Saver mode (e.g. if your battery is low), it may limit the visible framerate and using this addon may not appear to make animations smoother.",
```

Comment:
Google does officially use uppercase here, at least on the developer blog post I referenced.

Wonder if there are any other ways to make this more platform-neutral...

Reply:
I think Uppercase is fine in this case

Reply:
I prefer lowercase because the feature isn't called "Energy Saver" in every browser. Edge's "effeciency mode", for example, appears to be the same thing.